deb/common/rules

```diff
@@ -23,15 +23,7 @@ override_dh_auto_build:
 
 	# Build the compose plugin
 	cd /go/src/github.com/docker/compose \
-	&& GO111MODULE=on go mod download \
```

Comment:
I don't think the `go mod download` was needed (`go build` should do that already), but I can add it back if there's concerns.

Reply:
Automatically done on build afaik

Reply:
Yup; would only be beneficial if we would do this in a separate stage or `RUN` I guess, but currently build in this repo is done in `docker run`, so ..

Well, that'll all go away soon (hopefully!)